tests/experimental/metric_repository/test_column_descriptive_metrics_metric_retriever.py

```diff
@@ -33,6 +37,8 @@ def test_get_metrics():
         ("column.mean", "column=col2", ()): 2.7,
         ("column.median", "column=col1", ()): 2.5,
         ("column.median", "column=col2", ()): 2.7,
+        ("column_values.null.count", "column=col1", ()): 3,
+        ("column_values.null.count", "column=col2", ()): 5,
```

Comment:
Is it going to cause issues when the `null.count` is greater than the `table.row_count`?

Reply:
For this test it doesn't really matter since we are mocking the validator.compute_metrics() function with whatever we want, but I changed it because I think it is confusing!